vumi_twilio_api/twiml_parser.py

```diff
@@ -1,15 +1,47 @@
+import re
 import xml.etree.ElementTree as ET
 
 
 class Verb(object):
     """Represents a single verb in TwilML. """
+    name = "Verb"
 
-    def __init__(self, verb, attributes={}, nouns={}):
-        self.verb = verb
+    def __init__(self, attributes={}, nouns=[]):
         self.attributes = attributes
         self.nouns = nouns
 
 
+class Play(Verb):
+    """Represents the Play verb"""
+    name = "Play"
+    digits_re = re.compile('[0-9w]')
+
+    @classmethod
+    def from_xml(cls, xml):
+        """Returns a new Play Verb given an ElementTree object"""
+        nouns = [xml.text]
```

Comment:
`Nouns` is what it is called in the TwiML docs. For `Gather`, it is a list of `Say` and `Play` verbs. I think we can look at refactoring once we've looked at how all the verbs work, and see what is the best way to represent that.